tools/tck-api/src/main/scala/org/opencypher/tools/tck/api/Scenario.scala

```diff
@@ -158,7 +168,7 @@ case class Scenario(categories: List[String], featureName: String, number: Optio
             else Right(ctx)
 
           case (ctx, Parameters(ps, _)) =>
-            Right(ctx.copy(parameters = ps))
+            Right(ctx.withParameters(ps))
```

Comment:
To be consistent with my other comment below:

```suggestion
            Right(ctx.addParameters(ps))
```